boilerplate/_lib/ensure.sh

```diff
@@ -22,6 +22,14 @@ golangci-lint)
         curl -sfL "${DOWNLOAD_URL}" | tar -C "${GOPATH}/bin" -zx --strip-components=1 "golangci-lint-${GOLANGCI_LINT_VERSION}-${GOOS}-amd64/golangci-lint"
     fi
     ;;
+venv)
+    # Set up a python virtual environment
+    python3 -m venv .venv
```

Comment:
Where is this being created? I thought you didn't have permissions in the directory the user was running?

Reply:
The pwd is the repo root

Reply:
ah :+1:

Reply:
...based on having run this via a `make` from that directory.